public/src/layoutMode.js

```diff
@@ -0,0 +1,388 @@
+    tempBuffer.layout.title_x -= 5;
+}
+
+/**
+ * Helper functions to move the titles
+ * @category layoutMode
+ */
+export function toggleLayoutTitle() {
+    tempBuffer.layout.titleEnabled = !tempBuffer.layout.titleEnabled;
+}
+
+/**
+ * just toggles back to normal mode
+ * @category layoutMode
+ */
+function cancelLayout() {
+    if (layoutModeGet()) {
+        // eslint-disable-next-line no-use-before-define
+        toggleLayoutMode();
+    }
+}
+
+
+/**
+ * Store all data into layout and exit
+ * @category layoutMode
+ */
+function saveLayout() {
+    if (layoutModeGet()) {
+        for (let i = 0; i < tempBuffer.Input.length; i++) {
+            tempBuffer.Input[i].parent.layoutProperties.x = tempBuffer.Input[i].x;
+            tempBuffer.Input[i].parent.layoutProperties.y = tempBuffer.Input[i].y;
+        }
+        for (let i = 0; i < tempBuffer.Output.length; i++) {
+            tempBuffer.Output[i].parent.layoutProperties.x = tempBuffer.Output[i].x;
+            tempBuffer.Output[i].parent.layoutProperties.y = tempBuffer.Output[i].y;
+        }
+        globalScope.layout = { ...tempBuffer.layout };
+        // eslint-disable-next-line no-use-before-define
+        toggleLayoutMode();
+    }
+}
+
+/**
+ * Function to toggle between layoutMode and normal Mode
+ * the sidebar is disabled and n properties are shown.
+ * @category layoutMode
+ */
+export function toggleLayoutMode() {
+    if (layoutModeGet()) {
+        (layoutModeSet(false));
+        $('#layoutDialog').fadeOut();
+        globalScope.centerFocus(false);
+        dots();
+    } else {
+        (layoutModeSet(true));
+        $('#layoutDialog').fadeIn();
+        globalScope.ox = 0;
+        globalScope.oy = 0;
+        globalScope.scale = DPR * 1.3;
+        dots();
+        tempBuffer = new LayoutBuffer();
+        $('#toggleLayoutTitle')[0].checked = tempBuffer.layout.titleEnabled;
+    }
+    hideProperties();
+    update(globalScope, true);
+    scheduleUpdate();
+    // console.log(document.querySelector('#layoutDialog'))
+    $('#decreaseLayoutWidth').click(() => {
+        decreaseLayoutWidth();
+    });
+    $('#increaseLayoutWidth').click(() => {
+        increaseLayoutWidth();
+    });
+    $('#decreaseLayoutHeight').click(() => {
+        decreaseLayoutHeight();
+    });
+    $('#increaseLayoutHeight').click(() => {
+        increaseLayoutHeight();
+    });
+    $('#layoutResetNodes').click(() => {
+        layoutResetNodes();
+    });
+    $('#layoutTitleUp').click(() => {
+        layoutTitleUp();
+    });
+    $('#layoutTitleDown').click(() => {
+        layoutTitleDown();
+    });
+    $('#layoutTitleLeft').click(() => {
+        layoutTitleLeft();
+    });
+    $('#layoutTitleRight').click(() => {
+        layoutTitleRight();
+    });
+    $('#toggleLayoutTitle').click(() => {
+        toggleLayoutTitle();
+    });
+    $('#saveLayout').click(() => {
+        saveLayout();
+    });
+    $('#cancelLayout').click(() => {
+        cancelLayout();
+    });
```

Comment:
why not `$('#cancelLayout').click(cancelLayout);` ? 

we can avoid the extra wrap function for all the event bindings.